sensor/common/sensor/central_communication_test.go

```diff
@@ -2,32 +2,32 @@ package sensor
 
 import (
 	"context"
-	"net"
 	"path"
+	"strings"
 	"testing"
 	"time"
 
 	"github.com/stackrox/rox/generated/internalapi/central"
 	"github.com/stackrox/rox/generated/storage"
 	"github.com/stackrox/rox/pkg/centralsensor"
 	"github.com/stackrox/rox/pkg/concurrency"
-	"github.com/stackrox/rox/pkg/utils"
 	"github.com/stackrox/rox/sensor/common"
 	configMocks "github.com/stackrox/rox/sensor/common/config/mocks"
 	mocksDetector "github.com/stackrox/rox/sensor/common/detector/mocks"
 	"github.com/stackrox/rox/sensor/common/message"
+	mocksClient "github.com/stackrox/rox/sensor/common/sensor/mocks"
 	centralDebug "github.com/stackrox/rox/sensor/debugger/central"
```

Comment:
Remove this import along with `centralCommunicationSuite.fakeCentral` property.

Reply:
Oops. Done!